src/main/menu_main.cpp

```diff
@@ -2040,14 +2040,14 @@ void mainMenuDraw()
     {
         int i = 0;
         SuperPrint(g_mainMenu.controlsTitle, 3, MenuX, MenuY + 30*i++);
-#ifndef __ANDROID__
+#ifndef RENDER_FULLSCREEN_ALWAYS
         if(resChanged)
-            SuperPrint("WINDOWED MODE", 3, MenuX, MenuY + 30*i++);
+            SuperPrint("WINDOWED MODE", 3, 300, 350 + (30 * i++));
         else
```

Comment:
Best to roll these two lines back, they'll make a strange visual effect at non 800x600 resolutions

Reply:
Oh, I see now, didn't paid attention

Reply:
Fixed just now!